features/support/branch_helpers.rb

```diff
@@ -35,8 +35,8 @@ def branches_for_repository repository
 
 
 # Creates and pushes a branch
-def create_branch branch_name, remote: true
-  run "git checkout -b #{branch_name} main"
+def create_branch branch_name, remote: true, commit: 'main'
```

Comment:
I think this should be `start_point` which what the `git checkout` docs call it.
Any ref actually works here (a branch name, a tag, etc).